File: Sakila_Net/VB_SakilaLib/DAO/DAOCity.vb
'using System;
Imports  System.Data
'using System.Collections.Generic;
'using System.Configuration;
'using System.Collections;
Imports  SakilaLib.Cl.Harrys.Sakila.Model
'using Cl.Harrys.BD;

Namespace Cl
    Namespace Harrys
        Namespace Sakila
            Namespace DAO

                Public Class DAOCity
                    'Inherits SakilaLib.Cl.Harrys.Sakila.DAOBD.DAOCity
                    Public Shared  Function SelectAll(nPagina as Integer ,cantidad as Integer) as DataTable
                        Return DAOBD.DAOCity.SelectAll(nPagina, cantidad)	   
                    End Function
                    Public Shared  Function SelectOne( city as ClCity) as Boolean
                        Return DAOBD.DAOCity.SelectOne(city)     
                    End Function
                    Public Shared  Function SqlInsert( city as ClCity) as Integer
                        Return DAOBD.DAOCity.SqlInsert(city)     
                    End Function
                    Public Shared  Function SqlDelete( city as ClCity) as Integer
                        Return DAOBD.DAOCity.SqlDelete(city)     
                    End Function
                    Public Shared  Function SqlUpdate( city as ClCity) as Integer
                        Return DAOBD.DAOCity.SqlUpdate(city)     
                    End Function
                End Class

            End Namespace  
        End Namespace  
    End Namespace  
End Namespace  

File: Sakila_Net/Sakila_Net/CS_SakilaLib/DAO/DAOCity.cs
//using System;
using System.Data;
//using System.Collections.Generic;
//using System.Configuration;
//using System.Collections;
using Cl.Harrys.Sakila.Model;
namespace  Cl.Harrys.Sakila.DAO
{   
  public class DAOCity 
  {
       public static DataTable sqlSelectAll(int nPagina,int cantidad)
	   {
           return DAOBD.DAOCity.selectAll(nPagina,cantidad);
	   }
       public static bool sqlSelect(ClCity city)
	   {
            return DAOBD.DAOCity.SelectOne(city); 
	   }
	   public static int sqlInsert(ClCity city)
	   {
            return DAOBD.DAOCity.sqlInsert(city);
       }
	   public static int sqlDelete(ClCity city)
	   {
            return DAOBD.DAOCity.sqlDelete(city);
       }
	   public static int sqlUpdate(ClCity city)
	   {
            return DAOBD.DAOCity.sqlUpdate(city);
       }
   }
}
